Formula/swift.rb

```diff
@@ -81,50 +185,149 @@ def install
         clang-resource-headers compiler-rt clangd
       ]
 
-      args = %W[
+      on_macos do
+        llvm_components << "dsymutil"
+      end
+      on_linux do
+        swift_components += %w[
+          autolink-driver
+          sourcekit-inproc
+        ]
+        llvm_components << "lld"
+      end
+
+      pre_args = %W[
+        --host-cc=#{which(ENV.cc)}
+        --host-cxx=#{which(ENV.cxx)}
         --release --assertions
         --no-swift-stdlib-assertions
         --build-subdir=#{build}
-        --llbuild --swiftpm
+        --lldb --llbuild --swiftpm --swift-driver
```

Comment:
`swift-driver` is unused by the Swift 5.4 compiler, no need to build it.

Reply:
I'm recalling from memory here, but I don't think that's quite correct. It is used if you pass `SWIFT_USE_NEW_DRIVER` or something like that.

Since it's an optional feature, I'm happy to drop if it proves problematic. But we will have to solve any issues eventually when Swift 5.5 arrives.

Reply:
Yes, it is only used with Swift 5.4 if someone sets that environment variable, but it was somewhat buggy at 5.4 so I doubt anyone would.

And yeah, the plan appears to be to switch to it with 5.5, so you'll want to get this working by then. Luckily, it builds fine and the only error is the build script can't find `rsync` to copy it over to the install directory.